.buildkite/dagster-buildkite/dagster_buildkite/cli.py

```diff
@@ -16,6 +16,34 @@ def dagster():
     all_steps = dagit_steps()
     dagit_only = is_pr_and_dagit_only()
 
+    branch_name = os.getenv("BUILDKITE_BRANCH")
+    build_creator_email = os.getenv("BUILDKITE_BUILD_CREATOR_EMAIL")
+
+    if build_creator_email and build_creator_email.endswith("@elementl.com"):
+
+        if branch_name == "master" or is_release_branch(branch_name):
+            pipeline_name = "internal"
+            trigger_branch = branch_name
+            async_step = True
+        else:
+            pipeline_name = "oss-internal-compatibility"
```

Comment:
We can also control some of this in the Buildkite UI directly if we don't want to hardcode this in the OSS repo.